services/paymaster/src/rpc.rs

```diff
@@ -195,6 +196,110 @@ pub async fn send_and_confirm_transaction(
     confirm_transaction(pubsub, signature, Some(rpc.commitment())).await
 }
 
+/// Sends a transaction using the FTL service which handles both sending and confirmation.
+/// The FTL service requires a "confirm" parameter set to true in the RPC config.
+#[tracing::instrument(
+    skip_all,
+    fields(
+        tx_hash = %transaction.signatures[0],
+        result
+    )
+)]
+pub async fn send_and_confirm_transaction_ftl(
+    ftl_rpc: &RpcClient,
+    transaction: &VersionedTransaction,
+    config: RpcSendTransactionConfig,
+) -> Result<ConfirmationResultInternal, ErrorResponse> {
+    use base64::prelude::*;
+    use solana_rpc_client_api::request::RpcRequest;
+
+    let encoding = config.encoding.unwrap_or(UiTransactionEncoding::Base64);
+    let preflight_commitment = CommitmentConfig {
+        commitment: config.preflight_commitment.unwrap_or_default(),
+    };
+
+    let tx_bytes = bincode::serde::encode_to_vec(transaction, bincode::config::standard())
+        .map_err(|e| {
+            tracing::error!("Failed to serialize transaction: {e}");
+            ErrorResponse::from((
+                StatusCode::INTERNAL_SERVER_ERROR,
+                "Failed to serialize transaction",
+            ))
+        })?;
+    let tx_base64 = BASE64_STANDARD.encode(&tx_bytes);
+
+    let mut config_value = serde_json::to_value(RpcSendTransactionConfig {
+        encoding: Some(encoding),
+        preflight_commitment: Some(preflight_commitment.commitment),
+        ..config
+    })
+    .map_err(|e| {
+        tracing::error!("Failed to serialize config: {e}");
+        ErrorResponse::from((
+            StatusCode::INTERNAL_SERVER_ERROR,
+            "Failed to serialize config",
+        ))
+    })?;
+
+    // Add the "confirm" parameter to the config
+    if let Some(obj) = config_value.as_object_mut() {
+        obj.insert("confirm".to_string(), json!(true));
```

Comment:
so to make sure I understand, the only reason we can't use `ftl_rpc.send_transaction` is because we need to add this to `RpcSendTransactionConfig`?